spec/ace.md

```diff
@@ -264,9 +266,9 @@ Retrievable at `$AC_METADATA_URL/acMetadata/v1/pod`
 
 | Entry       | Description |
 |-------------|-------------|
-|annotations/ | Top level annotations from Pod Manifest. |
+|annotations/ | Top level annotations from Pod Manifest. Response body should conform to the sub-schema of the annotations property from the Pod specification (e.g. ```[ { "name": "ip-address", "value": "10.1.2.3" } ]```). |
```

Comment:
I think we should.   The apps need to support parsing JSON for the other endpoints -- I think it would be easier from the perspective of the application to make it consistent.